tf_training_args.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass, field
+
+from transformers import TrainingArguments
+
+
+# I am not sure, but maybe this shouldn't even be a child subclass,
+# but rather update TrainingArguments with some additional TF helpers.
+# Not sure about this...
+
+@dataclass
+class TFTrainingArguments(TrainingArguments):
+    strategy_name: str = field(default="onedevice")
+    optimizer_name: str = field(default="adam")
+    mode: str = field(default="classification")
```

Comment:
Could probably be an enum

Reply:
Definitely, each field can have its own enum. I'm also thinking pf reducing the number of properties, or at least find a way to have them in common with the PT Trainer as, indeed, they all can be part of the main TrainingArguments. The big picture would be to use the same conf file doesn't matter if we use the PT or TF trainer.

Reply:
As discussed, I'm fine with keeping a `TFTrainingArguments` for now.

if we need to define a mode here (still not 100% sure) I'd suggest we use "sequence-classification", "token-classification", etc.